examples/ex1.cpp

```diff
@@ -134,6 +139,10 @@ int main(int argc, char *argv[])
    b->AddDomainIntegrator(new DomainLFIntegrator(one));
    b->Assemble();
 
+   mesh->SetCurvature(1, false, -1, Ordering::byVDIM);
+   if (gpu) { config::Get().Cuda(true); }
+   if (pa)  { config::Get().PA(true);   }
```

Comment:
Partial assembly cannot be a global choice - it should be a parameter in `BilinearForm` or an operator format request through an `OperatorHandle`'s type in calls like `FormSystemMatrix` or `FormLinearSystem`.

Reply:
It could also be at the `Integrator` level...